### PR TITLE
New resource - `azurerm_express_route_port_authorization`

### PR DESCRIPTION
**Pull request by xuzhang3**
New resource - `azurerm_express_route_port_authorization`
```log
=== RUN   TestAccExpressRoutePortAuthorization_basic
--- PASS: TestAccExpressRoutePortAuthorization_basic (399.84s)
=== RUN   TestAccExpressRoutePortAuthorization_requiresImport
--- PASS: TestAccExpressRoutePortAuthorization_requiresImport (404.59s)
=== RUN   TestAccExpressRoutePortAuthorization_multiple
--- PASS: TestAccExpressRoutePortAuthorization_multiple (368.00s)
PASS
ok      github.com/hashicorp/terraform-provider-azurerm/internal/services/network       1209.603s
```

### CODE REVIEW

File: website/docs/r/express_route_port_authorization.html.markdown [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the ExpressRoute Port. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which to create the ExpressRoute Port. Changing this forces a new resource to be created.
+
```
Comment (katbyte) [inline website/docs/r/express_route_port_authorization.html.markdown:44]:
```suggestion
```

File: website/docs/r/express_route_port_authorization.html.markdown [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The ID of the ExpressRoute Port Authorization.
+
+* `authorization_key` - The Authorization Key.
+
+* `authorization_use_status` - The authorization use status.
```
Comment (katbyte) [inline website/docs/r/express_route_port_authorization.html.markdown:56]: could you expand on what this property does?

and does it make sense to be exposed in terraform?

**Review by katbyte — CHANGES_REQUESTED** [review]

File: website/docs/r/express_route_port_authorization.html.markdown (hunk 2 above)
Comment (xuzhang3) [reply website/docs/r/express_route_port_authorization.html.markdown:55]: This is the authorization status indicate if this authorization is in `use` or `available`.  This resource is similar to [azurerm_express_route_circuit_authorization](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/express_route_circuit_authorization)

Comment (katbyte) [reply website/docs/r/express_route_port_authorization.html.markdown:55]: 👍  thanks

**Review by katbyte — APPROVED** [review]
LGTM 🔐

**Pull request merged**